prover/src/zkevm/prover.rs

```diff
@@ -19,6 +19,7 @@ pub use outer_proof::AggCircuitProof;
 #[cfg(target_os = "linux")]
 extern crate procfs;
 
+#[allow(dead_code)]
```

Comment:
@lispc should we remove `OPT_MEM` here given that it's not used anywhere?

Reply:
ok.. it was a feature when OPT_MEM enabled, release super circuit pk before start agg circuit proving.   not needed now.

Reply:
Got it. So let's keep this line of code for now in case we may use it in the future?